validator/tests/test_client_request_handlers/mocks.py

```diff
@@ -143,18 +132,20 @@ def make_db_and_store(size=3, start='a'):
 
     return database, store, roots
 
-def make_store_and_cache(size=3):
+
+def make_store_and_tracker(size=3):
     """
     Creates and returns two related objects for testing:
         * store - a mock block store, with a default start
-        * cache - a batch cache with two batches, one in the store, and one not
+        * tracker - a batch tracker attached to the store, with one pending batch
+
     With defaults, the three block ids in the store will be:
         * 'B-0', 'B-1', B-2'
     The three batch ids in the store will be:
         * 'b-0', 'b-1', b-2'
-    And the two batch ids in the cache will be:
-        * 'b-2', 'b-3'
     """
-    store = MockBlockStore(size=size)
-    cache = MockBatchCache(start=str(size-1))
-    return store, cache
+    tracker = BatchTracker()
+    tracker.notify_pending('b-3')
```

Comment:
Meant to replace the docstring with the contents of the batch cache, with one with the contents of the tracker. It has been added back in.